xcconfig/xcconfig.go

```diff
@@ -2,34 +2,52 @@ package xcconfig
 
 import (
 	"fmt"
+	"path/filepath"
+	"strings"
+
 	"github.com/bitrise-io/go-utils/v2/fileutil"
 	"github.com/bitrise-io/go-utils/v2/pathutil"
-	"path/filepath"
 )
 
 // Writer ...
 type Writer interface {
-	Write(content string) (string, error)
+	Write(input string) (string, error)
 }
 
 type writer struct {
 	pathProvider pathutil.PathProvider
 	fileManager  fileutil.FileManager
+	pathChecker  pathutil.PathChecker
 }
 
 // NewWriter ...
-func NewWriter(pathProvider pathutil.PathProvider, fileManager fileutil.FileManager) Writer {
-	return &writer{pathProvider: pathProvider, fileManager: fileManager}
+func NewWriter(pathProvider pathutil.PathProvider, fileManager fileutil.FileManager, pathChecker pathutil.PathChecker) Writer {
+	return &writer{pathProvider: pathProvider, fileManager: fileManager, pathChecker: pathChecker}
 }
 
-func (w writer) Write(content string) (string, error) {
+func (w writer) Write(input string) (string, error) {
+	if w.isPath(input) {
+		pathExists, err := w.pathChecker.IsPathExists(input)
+		if err != nil {
+			return "", fmt.Errorf(err.Error())
+		}
+		if !pathExists {
+			return "", fmt.Errorf("provided xcconfig file path doesn't exist")
```

Comment:
```suggestion
			return "", fmt.Errorf("provided xcconfig file path doesn't exist: %s", input)
```

Let's include the actual path in the error message